about.html

```diff
@@ -9,22 +9,15 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
   <script src="https://kit.fontawesome.com/adf6e8ca6b.js" crossorigin="anonymous"></script>
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="css/style.css">
   <link rel="shortcut icon" href="images/title-ai.png">
   <title>About</title>
 </head>
 <body>
   <header>
-    <nav class="nav1">
-        <a class="nav-link" href="#" aria-label="facebook"><i id="facebook-icon" class="fa fa-facebook text-white"></i></a>
-        <a class="nav-link" href="#" aria-label="twitter"><i id="twitter-icon" class="fa fa-twitter text-white"></i></a>
-        <a class="nav-link" href="#">English</a>
-        <a class="nav-link" href="#">My Page</a>
-        <a class="nav-link" href="#">Logout</a>
-    </nav>
     <nav class="nav2">
-      <div class="navLogo"><a href="index.html"><img src="images/Logo.png" alt="logo"></a></div>
-      <div class="nav2Content">
+      <div class="navlogo"><a href="index.html"><img src="images/Logo.png" alt="logo"></a></div>
+      <div class="navcontent">
         <a class="nav-item" href="#">About</a>
         <a class="nav-item" href="#">Program</a>
         <a class="nav-item" href="#">sponsor</a>
```

Comment:
- Good job designing the navbar 💯 but there are still some issues to be resolved here:
- [ ] You are supposed to have 2 navbars, the top dark navbar with the social icons and the bottom light navbar with the page link. It would be good to implement both of them so as to follow the design guidelines 👍 
- [ ] (OPTIONAL) Also if you can make your `CC Campaign` button to be on a straight line, it would increase the user interface of your site.